Readme.md

```diff
@@ -82,3 +82,47 @@ Wegmeister:
       - 'Neos.Form.Builder:Password'
       - 'Neos.Form.Builder:PasswordWithConfirmation'
 ```
+
+## Cleanup commands
+
+The package comes with cleanup commands to delete data older than a date interval you can define in your settings.
+You can run the command manually or use a cron job.
+
+Add storages you wish to be cleaned up and define how long the data of each storage should be stored:
+
+```yaml
+
+Wegmeister:
+  DatabaseStorage:
+    cleanup:
+      # Add storage identifier you wish to be cleaned up
+      storageIdentifier1:
+        # Define how long the data should be stored as date interval
+        # https://www.php.net/manual/en/class.dateinterval.php
+        dateInterval: "P6M"
+        removeFiles: true
+      storageIdentifier2:
+          dateInterval: "P1Y"
+          removeFiles: false
+```
+
+Run the cleanup command for the configured storages:
+
+```
+./flow databasestorage:cleanupconfiguredstorages
+```
+
+You can also run a cleanup command for all existing storages. The command comes with parameters:
+
+| Parameter Name           | Data Type | Description                                                                                                                                                           |
+|--------------------------|-----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| date-interval            | string    | Defines which data should be deleted. We use the PHP DateInterval format. You can find more information [here](https://www.php.net/manual/en/class.dateinterval.php). |
+| skip-configured-storages | boolean   | If you have configured storages in your settings, you can skip them with this parameter.                                                                              |
```

Comment:
This should now be `include-configured-storages`, then we can merge 😊

Reply:
Good catch ... adjusted the docs